frontend/src/model/beta/releases/ReleaseForm.tsx

```diff
@@ -63,93 +69,112 @@ export default function ReleaseForm({
     onMinorReleaseChange(checked)
   }
 
+  useEffect(() => {
+    if (model && releases.length > 0) {
```

Comment:
This won't matter if you use the code from https://github.com/gchq/Bailo/pull/904/files#r1407761074, but there's still a learning point here.

The model isn't needed to get the latest release semver, so that isn't needed in this check.